com.ibm.wala.core/src/main/java/com/ibm/wala/analysis/pointers/BasicHeapGraph.java

```diff
@@ -376,97 +376,73 @@ public int compare(Object arg1, Object arg2) {
     }
   }
 
-  /*
-   * @see com.ibm.wala.util.graph.NumberedNodeManager#getNumber(com.ibm.wala.util.graph.Node)
-   */
+  /** @see com.ibm.wala.util.graph.NumberedNodeManager#getNumber(Object) */
```

Comment:
Hrm, I'm not sure these `@see` links to the superclass method add much value.  Are these something we should delete instead, under the assumption that a developer will go to go to superclass / interface methods for "inherited" documentation?

Reply:
This is not a `@see` link to the superclass method. `BasicHeapGraph` does not implement `NumberedNodeManager`.